src/web/View.tsx

```diff
@@ -50,6 +76,89 @@ export class View extends ViewBase<Types.ViewProps, {}> {
         isRxParentAText: React.PropTypes.bool.isRequired
     };
 
+    private resizeDetectorAnimationFrame: number;
+    private resizeDetectorNodes: {grow?: HTMLElement, shrink?: HTMLElement} = {};
```

Comment:
Nit: Coding convention for ReactXP calls for a space to the right of the opening curly brace and before the closing brace.